src/Calculator/Controls/EquationTextBox.cs

```diff
@@ -0,0 +1,719 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+using CalculatorApp;
+using CalculatorApp.Common;
+using CalculatorApp.Controls;
+using Windows.System;
+using Windows.Foundation;
+using Windows.ApplicationModel;
+using Windows.UI.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Media;
+
+namespace CalculatorApp
+{
+    namespace Controls
+    {
+        public sealed class EquationTextBox : Windows.UI.Xaml.Controls.Control
+        {
+
+            public EquationTextBox()
+            {
+            }
+
+
+            public Windows.UI.Xaml.Media.SolidColorBrush EquationColor
+            {
+                get { return (Windows.UI.Xaml.Media.SolidColorBrush)GetValue(EquationColorProperty); }
+                set { SetValue(EquationColorProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for EquationColor.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty EquationColorProperty =
+                DependencyProperty.Register(nameof(EquationColor), typeof(Windows.UI.Xaml.Media.SolidColorBrush), typeof(EquationTextBox), new PropertyMetadata(default(Windows.UI.Xaml.Media.SolidColorBrush)));
+
+
+            //public   DEPENDENCY_PROPERTY(Windows.UI.Xaml.Media.SolidColorBrush , EquationButtonForegroundColor);
```

Comment:
Could you please remove this meaningless comment since we have converted this dependency property?

Reply:
Removed